Kernel/Thread.cpp

```diff
@@ -1115,23 +1115,20 @@ DispatchSignalResult Thread::dispatch_signal(u8 signal)
         if (action.flags & SA_SIGINFO)
             fill_signal_info_for_signal(signal_info);
 
-#if ARCH(X86_64)
```

Comment:
This commit seems mis-ordered or mis-titled, it's claiming to make `dispatch_signal` work for aarch64, but it comes before 3 other commits that are required for it to work.

Reply:
Well, after that commit the code in `dispatch_signal` is executing correctly without hitting any FIXMEs, and it will then return to userspace, at which point it will jump to the `signal_trampoline`, and then is was crashing afterwards, since that was not implemented yet. But I'll re-order the commit anyway, because that makes more sense indeed :)